test/jasmine/tests/ternary_test.js

```diff
@@ -88,6 +88,22 @@ describe('ternary plots', function() {
             });
         });
 
+        it('should be able to restyle', function(done) {
+            Plotly.restyle(gd, { a: [[1,2,3]]}, 0).then(function() {
+                var transforms = [];
+                d3.selectAll('.ternary .point').each(function() {
+                    var point = d3.select(this);
+                    transforms.push(point.attr('transform'));
+                });
+
+                expect(transforms).toEqual([
```

Comment:
That's what I thought as well, but calcdata *was* being updated before the change, while the plot wasn't (unless I was bitten by the devtool watcher gremlin).